x-pack/plugins/secops/public/components/timeline/body/renderers/zeek_signature.tsx

```diff
@@ -97,13 +100,19 @@ export const Link = pure<{ value: string | null; link?: string | null }>(({ valu
     if (link != null) {
       return (
         <LinkFlexItem grow={false}>
-          <GoogleLink link={link}>{value}</GoogleLink>
+          <div>
+            <GoogleLink link={link}>{value}</GoogleLink>
+            <ExternalLinkIcon />
+          </div>
         </LinkFlexItem>
       );
     } else {
       return (
         <LinkFlexItem grow={false}>
-          <GoogleLink link={value} />
+          <div>
+            <GoogleLink link={value} />
+            <ExternalLinkIcon />
+          </div>
```

Comment:
A bit sad we have to add more DOM weight to these areas :-( but I assume it's for a good cause so 👍